### DISCUSSION TITLE
posts/python/locust

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#21] [General]
# posts/python/locust

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/python/locust.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2022-12-17T21:21:49.873Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Linux; Android 8.1.0; LM-Q710.FG) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Mobile Safari/537.36"

Just go go
